### PR TITLE
Add_granite_vision_model

### PR DESCRIPTION
**Pull request by Minestar6**
# Add granite3.2\_vision
添加了granite3.2\_vision模型的PaddlePaddle 实现，包含模型代码、配置文件、单元测试。
# 对齐验证
## 精度对齐
前 20 个 token 的 logits 对齐结果如下：
* mean diff:  5.409002e-05  
* max diff:   3.738403e-04
token完全一致

### 训练数据
文本数据
* GSM8K SFT 数据
* 训练步数：300 steps
granite_v
granite_diff

多模态数据

### 训练结果

vl_granite_v

vl_granite_diff

### CODE REVIEW

**Comment by risemeup1111** [comment]
## Failed CI看板

| 流水线名称 | 问题标签 | 修复建议 | 日志片段 |
| ------------ | ---------- | ---------- | ---------- |
| Clone-Linux / code-clone | 日志获取失败，根因不可判定 | 重新获取该 Job 日志并重跑 `code-clone`，确认是否为 CI 基础设施故障或代码克隆检测失败 | [Job](https://github.com/PaddlePaddle/PaddleFormers/actions/runs/33254549446/job/99117708417) |

<details>
<summary>日志分析报告</summary>

**失败的测试 case**:
```text
Job: Clone-Linux / code-clone
步骤: code-clone
关键错误: 日志获取失败: Client error '404 The specified blob does not exist.'
实际失败测试 case: 日志不可用，无法确认
```

**根本原因分析**:

当前快照仅确认 `code-clone` Job 状态为 failure，但其完整日志从 Actions 结果存储获取时返回 `404 The specified blob does not exist`。因此无法验证是 PR 中新增的 Llava-NeXT/SigLIP 代码触发了代码克隆检测，还是 CI 日志存储或抓取基础设施异常。

PR diff 本身不能替代 `code-clone` 的实际报错证据，当前不能据此认定任何具体代码行为为根因，也不能判断 CI 已通过。

**修复建议**:

1. 先恢复或重新抓取 Job `99117708417` 的日志，确认具体克隆检测规则、文件和重复代码位置。
2. 若日志仍无法获取，重跑 `Clone-Linux / code-clone`；若重跑成功，按新的检测结果修复对应代码重复问题。
3. 若重跑仍返回日志存储 404，应由 CI 基础设施维护者检查 Actions 日志归档和抓取服务。

</details>

Powered by Nyanpasu with gpt-5.6-luna 默认推理级别, please check the suggestions carefully.
